paseos/actors/actor_builder.py

```diff
@@ -305,6 +306,63 @@ def set_position(actor: BaseActor, position: list):
         actor._position = position
         logger.debug(f"Setting position {position} on actor {actor}")
 
+    @staticmethod
+    def set_cuboid_geometric_model(
+            actor: SpacecraftActor,
+            mass: float,
+            height: float,
+            length: float,
+            width: float,
+    ):
+        """Define geometry of the spacecraft actor.
+
+        Args:
+            actor (SpacecraftActor): Actor to update.
+            mass (float): Mass of the spacecraft in kg
+            height (float): Size of the spacecraft in z (nadir) direction in meters
+            length (float): Size of the spacecraft in y (along-track) direction in meters
+            width (float): Size of the spacecraft in x (across-track) direction in meters
+        """
+        assert mass >= 0, "Mass is <= 0"
+        assert height >= 0, "Height is <= 0"
+        assert length >= 0, "Length is <= 0"
+        assert width >= 0, "Width is <= 0"
+
+        actor._mass = mass
+        geometric_model = CuboidGeometricModel(
+            local_actor=actor,
+            actor_mass=mass,
+            actor_height=height,
+            actor_length=length,
+            actor_width=width,
+        )
+        actor._moi = geometric_model._find_moi
+
+    @staticmethod
+    def set_geometric_model_from_import(
+            actor: SpacecraftActor,
+            mass: float,
+            file_name: str,
+    ):
+        """Define geometry of the spacecraft actor.
+
+        Args:
+            actor (SpacecraftActor): Actor to update.
+            mass (float): Mass of the spacecraft in kg
+            file_name (string): Name of the .obj file to use as geometric mesh
+        """
+        assert mass >= 0, "Mass is <= 0"
+
+        actor._mass = mass
+        geometric_model = ImportGeometricModel(
+            local_actor=actor,
+            actor_mass=mass,
+            model_name=file_name
+        )
+        actor._moi = geometric_model._find_moi
+        actor._center_of_gravity = geometric_model.find_cg()
+        actor._mesh = geometric_model._mesh
```

Comment:
as discussed in #191 let's avoid file IO and just pass `(vertices: np.array, faces: np.array, scale;int)` ?

Reply:
Yeah, I've implemented these changes based on the suggestion in #191